tests/integration/data/legacy/search_objects/case-06-response.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "xyz",
-    "version": "1.1",
+    "version":  "1.1",
```

Comment:
Was the intentional?

Reply:
No, I did try to catch any extra spaces, since I know the code quality checks will flag them. And no, I don't know how an extra space just crept in there!

Reply:
:-)